tests/functional/ext/private_import/private_import.py

```diff
@@ -18,8 +18,6 @@
 # Ignore dunders
 import __asd__
 import __future__
-from __future__ import print_function
-from __future__ import __print_function__
```

Comment:
We should probably replace by something similar here, it feel like it's part of what is tested.

Reply:
Perhaps best to revert this one; it does indeed look like it is useful for the private import checking.

Reply:
Yeah sure, we could update the import to something more modern but there's no real reason to do that.